L1Trigger/TrackTrigger/python/TTStubAlgorithmRegister_cfi.py

```diff
@@ -2,41 +2,55 @@
 
 # First register all the hit matching algorithms, then specify preferred ones at end.
 
-# The stub windows used here refer to the definition provided in this link:
+# The stub windows used has been optimized for for PU200 events
+# We use by default the tight tuning
 #
-# http://sviret.web.cern.ch/sviret/docs/CMS/SLHC/L1TT_XX1215.pdf
+# Definition is presented here:
 #
-# Extension to the tilted geometry is discussed here:
-#
-# https://indico.cern.ch/event/536881/contributions/2219856/
+# https://indico.cern.ch/event/681577/#4-update-of-the-track-trigger
 #
 # This script is adapted to the very last Tilted Tracker geometry to date (tracker T5)
-#
+# This version was tested on CMSSW 10_0_0
```

Comment:
10_0_0 isn't out yet... do you mean a prerelease?

Reply:
Yes, was meaning pre1